src/tests/test_utils.cpp

```diff
@@ -466,6 +470,42 @@ class Utility_Function_Tests final : public Test {
          result.test_is_eq(Botan::store_le<std::vector<uint8_t>>(TestEnum32::_1), Botan::hex_decode("78563412"));
          result.test_is_eq<std::array<uint8_t, 4>>(Botan::store_be(TestEnum32::_2), {0x78, 0x56, 0x34, 0x12});
 
+#if defined(BOTAN_HAS_SIMD_32)
+
```

Comment:
You also have to guard this at runtime with `CPUID::has_simd_32` since it's quite possible (if somewhat less likely these days) to compile with SIMD enabled but it's not available at runtime on whatever CPU we land on.